### PR TITLE
Update cross-toolchain to produce zip instead of tar.xz

### PR DESCRIPTION
**Pull request by usercw88**
WI: AB#2788360

Testing:

- [ X ] Executed script locally and verified zip was produced instead of tar.xz

### CODE REVIEW

File: ni-oe-init-build-env [hunk 1]
```diff
@@ -130,7 +131,7 @@ export BB_ENV_PASSTHROUGH_ADDITIONS="${BB_ENV_PASSTHROUGH_ADDITIONS:-} \
 ## PYREX VARIABLES ###
 # Setup pyrex config variables
 export PYREXCONFFILE=${PYREXCONFFILE:-"${SCRIPT_ROOT}/pyrex.ini"}
-export PYREX_CONFIG_BIND="${NILRT_ROOT}"
+export PYREX_CONFIG_BIND="${NILRT_ROOT} ${HOME}/.gitconfig"
```
Comment (chaitu236) [inline ni-oe-init-build-env]: We shouldn't add this

**Review by chaitu236 — CHANGES_REQUESTED** [review]
In the commit message, we don't usually add WI number but instead describe the changes.

It is enough to mention WI number in PR description which you've already done.

**Comment by chaitu236** [comment]
> In the commit message, we don't usually add WI number but instead describe the changes.
> 
> It is enough to mention WI number in PR description which you've already done.

PS: Can you also start first line of commit with the filename e.g., https://github.com/ni/nilrt/commit/3680c6a675f3c4366a494a50b44be54e5c321960.

**Review by amstewart — APPROVED** [review]

**Review by chaitu236 — APPROVED** [review]

**Comment by chaitu236** [comment]
Cherry-picked to nilrt/master/next

**Pull request merged**